### DISCUSSION TITLE
Random aliases setting is overriding browser extension settings

### DISCUSSION THREAD

**Discussion by PeckinPaw** [simple-login/app#2223] [Q&A]
I have the browser extension installed, and I have the settings set to "Include website address in one-click alias creation on browser extension". However, when I click on the SimpleLogin extension's icon in an email address box on a web page, the generated alias email address that gets created is using the settings from the Aliases section of the setting page (Random word, my domain, random word from dictionary). 

How do I get the browser extension to generate the alias email in the proper format? (Website Address, My Domain).

**Comment by nguyenkims** [comment]
How do you set the "Include website address in one-click alias creation on browser extension" setting? This setting can only be set on https://app.simplelogin.io/dashboard/setting and will be used by all SL apps.

**Reply by PeckinPaw** [reply]
I have the setting turned on but it's not working.
